en/docs/references/performance-analysis.md

```diff
@@ -1,18 +1,77 @@
 # Performance Analysis
 
-The Performance Analyzer tool is an AI-based tool that generates intelligent estimations of the performance of integration-based services and APIs based on historical data. To generate the forecasts, it uses advanced machine learning techniques.
-Making accurate performance estimations before releasing an application you've developed is important due to the following reasons:
+The Performance Analyzer tool is an AI-based tool that generates intelligent estimations of the performance of integration-based services and APIs based on historical data. The Performance Analyzer tool allows developers to obtain forecasts at development time using advanced machine learning techniques and mathematical performance models.
 
-- When you create service-level agreements (SLAs) for your application, an accurate forecast of performance allows you to provide valuable insights relating to the scalability of the application and the ways to optimize its performance.
-- You can avoid missing important performance characteristics as a result of combining multiple applications including external ones.
+Traditionally, performance analysis includes performance debugging which involves measuring the performance of services by load testing them. The Performance Analyzer allows you to measure performance at the application development stage, significantly reducing the time and cost spent on performance debugging.
+
+A component may have more than one execution path serving requests. The Performance Analyzer tool can analyze all the execution paths and determine the **performance-critical path**: the path that is estimated to take the longest time to process a single request.
+
+## Advantages of using the Performance Analyzer
+
+Making accurate performance estimations before releasing an application you have developed is important due to the following reasons:
+
+- You can identify the performance-critical path of a component at development time. 
+- The forecasts you obtain for individual connector actions help identify any bottlenecks in each execution path.
+- You can detect any performance anti-patterns and low-quality code in your component implementation that impacts the overall performance. 
+- When you create service-level agreements (SLAs) for your application, an accurate performance forecast allows you to provide valuable insights relating to the scalability of the application and the ways to optimize its performance.
+- You can avoid missing significant performance characteristics when combining multiple applications, including external applications.
 - You can correctly assess the cost and feasibility of running the application in a production environment.
+- You can reduce the time and cost spent on debugging performance issues by eliminating or minimizing iterative load testing efforts.
+
+## Use the Performance Analyzer
```

Comment:
Can you reword this topic?

Reply:
what would be the message you would want it to give?

Reply:
'How to use' or 'A guide to use' sounds better in my opinion.

Reply:
We use a style in our headings which we start with a verb.